src/lib/eco-router/api.ts

```diff
@@ -109,9 +109,32 @@ export async function getExactIn(
       })
   })
 
+  // Gnosis Protocol V2
+  const gnosisProtocolTrade = new Promise<GnosisProtocolTrade | undefined>(async resolve => {
+    if (!RoutablePlatform.GNOSIS_PROTOCOL.supportsChain(chainId as ChainId)) {
+      return resolve(undefined)
+    }
+
+    GnosisProtocolTrade.bestTradeExactIn({
+      currencyAmountIn,
+      currencyOut,
+      maximumSlippage,
+      receiver,
+    })
+      .then(resolve)
+      .catch(error => {
+        resolve(undefined)
+        console.log(error)
```

Comment:
Suggestion: use `.error` instead of `.log`

Reply:
agreed, done